boards/yellow/yellow.go

```diff
@@ -0,0 +1,160 @@
+package yellow
+
+import (
+	"github.com/godbus/dbus/v5"
+	"github.com/godbus/dbus/v5/introspect"
+	"github.com/godbus/dbus/v5/prop"
+
+	"github.com/home-assistant/os-agent/utils/bootfile"
+	logging "github.com/home-assistant/os-agent/utils/log"
+)
+
+const (
+	objectPath = "/io/hass/os/Boards/Yellow"
+	ifaceName  = "io.hass.os.Boards.Yellow"
+	bootConfig = "/mnt/boot/config.txt"
+)
+
+var (
+	optLEDPower     bool
+	optLEDDisk      bool
+	optLEDHeartbeat bool
+)
+
+type yellow struct {
+	conn  *dbus.Conn
+	props *prop.Properties
+}
+
+func getStatusLEDPower() bool {
+	value, _ := bootfile.ReadOption(bootConfig, "dtparam=pwr_led_trigger", "on")
```

Comment:
The trigger is used as `linux,default-trigger`, `on` is not a valid trigger.

I think we need to check the reverse here: If it is `none`, then the LED is disabled.

Reply:
Well, that is just a soft default because not exists meaning its on